src/test/java/seedu/address/model/NationalityBookTest.java

```diff
@@ -22,7 +22,7 @@
 import seedu.address.model.person.exceptions.DuplicatePersonException;
 import seedu.address.testutil.PersonBuilder;
 
-public class AddressBookTest {
+public class NationalityBookTest {
```

Comment:
`NationalityBookTest` → `AddressBookTest` as `AddressBook` ≠ `Address`
Same for the java file name.